src/App.js

```diff
@@ -1,5 +1,21 @@
-function App() {
-  return <p>Hello World!</p>
+import BookList from './components/BookList'
+import styled from 'styled-components'
+
+function App({ data }) {
+  return (
+    <main>
+      <Header>
+        <h2>List of Books</h2>
+        <h3>Hi user!</h3>
+        <p>Welcome back! Here is your booklist:</p>
+      </Header>
```

Comment:
I think this could be an `h1` and `h2` for better structure of your page.
```suggestion
      <Header>
        <h1>List of Books</h1>
        <h2>Hi user!</h2>
        <p>Welcome back! Here is your booklist:</p>
      </Header>
```